input/default.toml

```diff
@@ -115,9 +115,8 @@ author  = "Harrison Nicholls, Tim Lichtenberg"
 # Planetary structure - physics table
 [struct]
 
-    # Enclosed mass and radius at the atmosphere-interior interface
+    # Enclosed mass at the atmosphere-interior interface
```

Comment:
Should this be "enclosed mass at the top of atmosphere" according to our discussion earlier on?

Reply:
Ah yes - I forgot to make this change! It should indeed. How about I make a new issue for this to treat it separately, since this PR is already too large? We can then also add the option of passing the radius instead of the mass.

Reply:
Good for me!